### PR TITLE
Add pre-populated schema

### LINKED ISSUE #3594: Pre-populate the Schemas
**Describe the solution you'd like**
Schema editor: pre-populate the text boxes with some better defaults. Currently, the editor starts with empty text boxes.

### PR DESCRIPTION
**Pull request by jzwang43**
### Reasons for making this change

Pre-populate the text boxes with some better defaults. Currently, the editor starts with empty text boxes.

Fixes #3594

### Screenshots

a

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: frontend/src/constants.js [hunk 1]
```diff
@@ -60,3 +60,28 @@ export const DEFAULT_POST_PROCESSOR = {
     size: 'size',
     date: 'date',
 };
+
+export const PRE_POPULATE_SCHEMA_ROWS = [
```
Comment (epicfaace) [inline frontend/src/constants.js]: consider uniformity with https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/worksheet_util.py#L575

Comment (jzwang43) [reply frontend/src/constants.js]: @epicfaace Thanks I updated the fields so it's the same as the default ones.

Comment (epicfaace) [reply frontend/src/constants.js]: Sounds good. Can we rename to `DEFAULT_SCHEMA_ROWS`?

Comment (epicfaace) [reply frontend/src/constants.js]: Also add a comment describing what this is used for.

Finally, add a comment here saying that it should be synced with https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/worksheet_util.py#L575, and a comment in https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/worksheet_util.py#L575 saying it has to be synced with this file.

Comment (jzwang43) [reply frontend/src/constants.js]: Thanks I have renamed the constant and updated the comments.

**Review by epicfaace — APPROVED** [review]

**Pull request merged**